backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -431,15 +431,6 @@ public Parcel PendingUpdate(Parcel parcel)
                 foreach (var parcelBuilding in originalParcel.Buildings)
                 {
                     var updateBuilding = parcel.Buildings.FirstOrDefault(pb => pb.BuildingId == parcelBuilding.BuildingId);
-                    if (updateBuilding == null)
```

Comment:
as far as I'm aware, there is no FE workflow that allows a user to **disassociate** one or more buildings from a parcel. So even though this code is correct, it serves no purpose other then to potentially harm the land->building relationships if the frontend makes a bad request.

Reply:
Also worth noting that I've left in the inverse of this logic (if a user submits a building with fewer associated land than what is in the DB, all **leftover** associations will be removed. I've done that because there is a FE workflow to remove associated land from a building.